src/app/components/help/HelpContent/HelpContent.tsx

```diff
@@ -25,34 +25,59 @@ const HelpContent: React.FC = () => {
-          <li>Buiten kantoortijden Datapunt, { showOptionalProperty(contacts[1]?.name) } <PhoneLink phoneNumber={ showOptionalProperty(contacts[1]?.phone_number) } /></li>
-        </ul>
       </RowWithColumn>
       <RowWithColumn>
-        <Heading as="h2">Vraag een demo aan</Heading>
-        <Paragraph>
-        Ben je nieuw met { title }, loop je ergens tegen aan of wil je gewoon graag persoonlijke uitleg over { title }? Neem dan contact op met { showOptionalProperty(contacts[2]?.name) }, { showOptionalProperty(contacts[2]?.title) }.
-        </Paragraph>
-        <ul>
-          <li>Telefonisch: te bereiken tijdens kantooruren op ma, di, wo en do op <PhoneLink phoneNumber={ showOptionalProperty(contacts[2]?.phone_number) } /></li>
-          <li>Per e-mail: <EmailLink email={ showOptionalProperty(contacts[2]?.email) } /></li>
-        </ul>
-        <Paragraph>
-        Ook voor goede ideeën en suggesties <span role="img" aria-label="Slightly Smiling Face">🙂</span>
-        </Paragraph>
+        <Heading as="h2">Bij wie kan ik terecht met vragen?</Heading>
+      </RowWithColumn>
+      <RowWithColumn>
+        <AccordionWrapper>
+          <Accordion title="Werkproces">
+            <Paragraph>
+              Voor het werkproces, oftewel hoe { titleShort } te gebruiken, is een aparte werkinstructie geschreven. Deze kan je opvragen bij je teamleider. De teamleider is er verder ook om vragen en suggesties betreft het werkproces te beantwoorden.
+            </Paragraph>
+          </Accordion>
+          <Accordion title="Algemeen gebruik zaaksysteem">
+            <Paragraph>
+              In de factsheet, waar bovenaan deze pagina naar verwezen wordt, lees je wat je in { titleShort } kan vinden. Heb je specifiekere vragen hoe { titleShort } werkt of kom je ergens niet uit, neem dan contact op met een van de key-users. Dit zijn directe collega’s die veel kennis hebben van { titleShort } en hier speciaal voor getraind zijn. Zij helpen je graag verder. Vraag aan je teamleider bij wie jij het beste terecht kan.  
+            </Paragraph>
+          </Accordion>
+          <Accordion title="Technisch support" >
+            <Paragraph>
+              Werkt { titleShort } niet (goed)? Probeer de pagina opnieuw te laden door op het refresh icoon <RefreshIcon /> te klikken.
+            </Paragraph>
+            <Paragraph>
+              Werkt { titleShort } dan nog steeds niet, neem dan contact op met:
+              <ul>
+                <li>Tijdens kantooruren ontwikkelteam { titleShort }, { showOptionalProperty(contacts[0]?.name) } <PhoneLink phoneNumber={ showOptionalProperty(contacts[0]?.phone_number) } /></li>
+                <li>Buiten kantoortijden Datapunt, { showOptionalProperty(contacts[1]?.name) } <PhoneLink phoneNumber={ showOptionalProperty(contacts[1]?.phone_number) } /></li>
```

Comment:
This could create an `<a href="(onbekend)">(onbekend)</a>`. Maybe `contacts[1]?.phone_number ? <PhoneLink phoneNumber={ contacts[1]?.phone_number }> : showOptionalProperty()` would be nicer.